tools/c7n_azure/tests_azure/tests_resources/test_networksecuritygroup.py

```diff
@@ -121,12 +132,15 @@ def test_deny_port(self):
                  'value_type': 'normalize',
                  'value': 'c7n-nsg'},
                 {'type': 'ingress',
-                 'ports': '8086',
+                 'ports': '22',
```

Comment:
This test was the one that caused me to catch that the filter wasn't respecting the rule with the highest priority. The rule that denies access to 8086 is superseded by a higher-priority one that allows it. The rule that denies 22 is not superseded.